Core/src/IfcGeometryConverter/SplineConverter.h

```diff
@@ -39,7 +39,13 @@ namespace OpenInfraPlatform
 			class SplineConverterT : public ConverterBaseT<IfcEntityTypesT> // TODO 2020.04.08.: spline converter does not apply length corrections
 			{
 				public:
-					SplineConverterT() {}
+					SplineConverterT(
+						std::shared_ptr<GeometrySettings> geomSettings,
+						std::shared_ptr<UnitConverter<IfcEntityTypesT>> unitConverter)
+						:
+						ConverterBaseT<IfcEntityTypesT>(geomSettings, unitConverter) 
+					{}

```

Comment:
The pointer to PlacementConverterT was added in Commit 9078d7d

Reply:
If the placementConverter is not needed anywhere in the SplineConverter, then there is no need to have it.